apps/www/data/features.tsx

```diff
@@ -2036,4 +2036,35 @@ This feature is especially beneficial for Python developers looking to build rob
       availableOnSelfHosted: true,
     },
   },
+  {
+    title: 'OrioleDB',
+    subtitle: "New Postgres storage engine that's better than Heap storage.",
+    description: `
+OrioleDB is a PostgreSQL storage extension built on its pluggable storage framework. Serving as a direct replacement for PostgreSQL's Heap storage, it addresses scalability challenges while harnessing the full power of modern hardware. Designed to integrate effortlessly with PostgreSQL, OrioleDB enhances performance, efficiency, and scalability, all while maintaining the reliability and robustness PostgreSQL users depend on.
+
+## Key benefits:
+1. Seamless Integration: A drop-in replacement for PostgreSQL’s Heap storage, enabling easy adoption without major changes to existing workflows.
```

Comment:
```suggestion
1. Fully Integrated: A drop-in replacement for PostgreSQL’s Heap storage, enabling easy adoption without major changes to existing workflows.
```
nit: don't want to call it seamless until all indexes are supported